src/applications/disability-benefits/all-claims/tests/pages/toxicExposure/gulfWar1990Locations.unit.spec.jsx

```diff
@@ -109,4 +109,34 @@ describe('Gulf War 1990 Locations', () => {
     await userEvent.click(getByText('Submit'));
     expect($('va-checkbox-group').error).to.equal(noneAndLocationError);
   });
+
+  it('should submit with `notsure` and other locations selected', async () => {
+    const onSubmit = sinon.spy();
+
+    const { container, getByText } = render(
+      <DefinitionTester
+        schema={schema}
+        uiSchema={uiSchema}
+        data={{}}
+        onSubmit={onSubmit}
+      />,
+    );
+
+    const checkboxGroup = $('va-checkbox-group', container);
+    await checkboxGroup.__events.vaChange({
+      target: { checked: true, dataset: { key: 'oman' } },
+      detail: { checked: true },
```

Comment:
there's some new test utils you can use here in `src/applications/mhv-secure-messaging/util/testUtils.js`

Reply:
@christinec-fftc When trying to use the new `selectVaSelect` function from `src/applications/mhv-secure-messaging/util/testUtils.js`, this linter error is appearing:
```
No cross app imports allowed: disability-benefits importing from mhv-secure-messagingeslint(@department-of-veterans-affairs/no-cross-app-imports)
```

It seems like we can forgo using this for now, while that rule is in place. Do you agree?

Reply:
I was able to use the new `checkVaCheckbox` method you added to `src/applications/disability-benefits/all-claims/tests/testUtils.js` instead